libraries/mbed/targets/hal/TARGET_RENESAS/TARGET_RZ_A1H/serial_api.c

```diff
@@ -512,7 +512,11 @@ int serial_getc(serial_t *obj) {
     int data;
     int was_masked;
 
+#if defined ( __ICCARM__ )
+    was_masked = __disable_irq_iar();
```

Comment:
The above changes is included to this pull request.
In addition, after we compiled with the IAR and confirmed the operation, we did a pull request.

libraries/mbed/targets/cmsis/core_caFunc.h
![temp1](https://cloud.githubusercontent.com/assets/9958200/13138695/f0ac45da-d66b-11e5-87c5-19459de9d8e8.PNG)
![temp2](https://cloud.githubusercontent.com/assets/9958200/13138791/73af3352-d66c-11e5-9f07-772a4442354f.PNG)
![temp3](https://cloud.githubusercontent.com/assets/9958200/13138793/73b02ece-d66c-11e5-9a3c-7f0d3698402f.PNG)
![temp4](https://cloud.githubusercontent.com/assets/9958200/13138792/73afce5c-d66c-11e5-9266-c5fb61f13d4c.PNG)
![temp5](https://cloud.githubusercontent.com/assets/9958200/13138795/73b3266a-d66c-11e5-99fe-1116b1b14604.PNG)
![temp6](https://cloud.githubusercontent.com/assets/9958200/13138794/73b0ee18-d66c-11e5-8244-69eed430a6ae.PNG)